File: test/fixtures/stdlib/string/trim.expected.py
(lambda _: kAssert("  hello  ".strip() == "hello"))(None)
(lambda _: kAssert("no spaces".strip() == "no spaces"))(None)
(lambda _: kAssert("   ".strip() == ""))(None)
(lambda _: kAssert((" hello " + " world ").strip() == "hello  world"))(None)

File: test/fixtures/stdlib/string/transform/trim.expected.js
(function(_) { return (function() { if (!("  hello  ".trim() == "hello")) throw new Error("Assertion failed"); return true; })(); })(null);
(function(_) { return (function() { if (!("no spaces".trim() == "no spaces")) throw new Error("Assertion failed"); return true; })(); })(null);
(function(_) { return (function() { if (!("   ".trim() == "")) throw new Error("Assertion failed"); return true; })(); })(null);
(function(_) { return (function() { if (!((" hello " + " world ").trim() == "hello  world")) throw new Error("Assertion failed"); return true; })(); })(null);
